DASCharter-2021.html

```diff
@@ -1160,6 +1160,13 @@ <h3>
           <dd>
             Coordinate on accessibility of user interface features implied by the APIs.
           </dd>
+          <dt>
+            <a href="https://www.w3.org/2011/audio/">Audio Working Group</a>
+          </dt>
+          <dd>
+            Coordinate on the <a href="https://oyiptong.github.io/compute-pressure/">Compute Pressure
+            </a> API to make sure the API can meet the requirements in real-time audio applications.
```

Comment:
I'm not sure the goal is to ensure the API meets the requirements for real-time audio - there may be valid reasons for two different APIs if the use cases differ enough. My AC review comment was intended simply to flag the related work happening in Audio WG and therefore need for cross-group awareness and discussion.

Reply:
We can also keep this relationship informal and not codify it in the charter text. WDYT?

Reply:
it doesn't hurt to have it

Reply:
I’m of course more than happy to receive a proposal for @chrisn for a more accurate description of the key coordination points between the groups.

Reply:
How about something like this?
> Coordinate on API requirements for observing CPU utilization to guide the development of either a common solution, or complementary solutions, for general compute and real-time audio use cases